include/vcpkg/base/message-data.inc.h

```diff
@@ -14,10 +14,6 @@ DECLARE_MESSAGE(AString, (), "", "a string")
 DECLARE_MESSAGE(ADateVersionString, (), "", "a date version string")
 DECLARE_MESSAGE(AddArtifactOnlyOne, (msg::command_line), "", "'{command_line}' can only add one artifact at a time.")
 DECLARE_MESSAGE(AddCommandFirstArg, (), "", "The first parameter to add must be 'artifact' or 'port'.")
-DECLARE_MESSAGE(AddFirstArgument,
```

Comment:
This part of the change seems incorrect. In particular, `find zlib` will now print "The first parameter to add must be 'artifact' or 'port'.". If anything, AddCommandFirstArg should die in favor of AddFirstArgument?